### DISCUSSION TITLE
I have a student account, but i dont have a button "You're eligible to use copilot for free"

### DISCUSSION THREAD

**Discussion by ikeitw** [community/community#122177] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Github verified my account as student, but i cant claim Copilot for free, how can i fix this?

**Comment by Akash1134** [comment, marked as answer]
Hi @gho228 ,

Please [refer this guide](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279) to start setting up GitHub student and GitHub Copilot as an authenticated student developer.

For more details, see [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) 

Thanks!

**Comment by ikeitw** [comment]
Thanks for the guide
